### PR TITLE
Demo branch

### PR DESCRIPTION
**Pull request by SerineBetraoui**
:)

### CODE REVIEW

**Review by Aayush-Srikanth — APPROVED** [review]
looks good to me

File: src/test/java/RestAssuredAPITest.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+public class RestAssuredAPITest {
+
+    @Test
+    public void GetRequest() {
+        RestAssured.baseURI = "http://3.11.77.136/index.php";
+        RequestSpecification httpRequest = RestAssured.given();
+        Response response = httpRequest.request(Method.GET, "");
+        System.out.println("Status received => " + response.getStatusLine());
+        System.out.println("Response=>" + response.prettyPrint());
```
Comment (Leosaw) [inline src/test/java/RestAssuredAPITest.java:15]: Good to go

**Review by jacktegart — COMMENTED** [review]
noice

File: src/test/java/RestAssuredAPITest.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import io.restassured.RestAssured;
+import io.restassured.http.Method;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import org.junit.jupiter.api.Test;
+
+public class RestAssuredAPITest {
+
+    @Test
+    public void GetRequest() {
```
Comment (SuzyLady) [inline src/test/java/RestAssuredAPITest.java:10]: fancy

**Review by SuzyLady — APPROVED** [review]

**Review by Leosaw — APPROVED** [review]

File: src/test/java/RestAssuredAPITest.java (hunk 1 above)
Comment (SerineBetraoui) [reply src/test/java/RestAssuredAPITest.java:15]: B)

**Pull request merged**